config/routes/api.rb

```diff
@@ -1,4 +1,6 @@
 Openfoodnetwork::Application.routes.draw do
+  mount DfcProvider::Engine, at: '/'
```

Comment:
I think we still want a feature toggle (as discussed in previous comments), otherwise we're unconditionally enabling this in production for all instances here.

Reply:
Actually, there is a constraint dealing with the feature switch behind these routes, it is inside the engine, `DfcProvider::ActivationConstraint`.